### PR TITLE
Fix python problems discovered by flake8

### PR DESCRIPTION
**Pull request by clumens**
The first patch got a little out of control, but it calms down after that.  Most of this is just style stuff, though there's a couple more reasonable things in there too.  With this, the python code passes both pylint and flake8.

### CODE REVIEW

File: python/Makefile.am [hunk 1]
```diff
@@ -24,3 +24,13 @@ check-local:
 .PHONY: pylint
 pylint:
 	pylint $(SUBDIRS)
+
+# Disabled warnings:
+# E501 - Line too long
+# E722 - Do not use bare except
+# F401 - Module imported but unused
```
Comment (kgaillot) [inline python/Makefile.am:31]: seems like we should be able to fix those last two

**Review by kgaillot — COMMENTED** [review]
Nice. Update copyrights (BTW we recently added `make -C devel copyright`)

File: python/Makefile.am (hunk 1 above)
Comment (clumens) [reply python/Makefile.am:31]: We're using bare except in three places, two of which I think we probably want to catch everything - run a test and if anything goes wrong stop profiling; and run a scenario and if anything goes wrong clean up.  The third could be fixed.  The unused import is for stuff like `pacemaker/__init__.py` where we are re-exporting things.

Comment (clumens) [reply python/Makefile.am:31]: Added a patch to address the bare export (catch the one exception we care about in one place, and ignore the other two with a pragma), and updated the Makefile.am patch to only ignore unused imports in `__init__.py` files.

**Review by kgaillot — APPROVED** [review]
Looks good. Needs rebase

**Comment by clumens** [comment]
Rebased

**Pull request merged**